torch/csrc/jit/codegen/cuda/utils.h

```diff
@@ -58,6 +58,16 @@ class PolymorphicBase {
     return downcast_ptr;
   }
 
+  // Check if the runtime time is T (or derived from T)
+  //
+  // NOTE: Don't use this for conditional casts. Use:
```

Comment:
Why is this? Multiple calls to `dynamic_cast`?

Reply:
right, in debug builds we'd have a double dynamic_cast.

but even in release builds, it's more natural to use dynamic_cast<T> in this case since that's exactly what it does (no point in reinventing a different way of doing what is a basic language feature)

Reply:
Thanks. Makes sense.